metricflow/test/integration/test_cases/itest_metrics.yaml

```diff
@@ -1312,4 +1312,107 @@ integration_test:
         GROUP BY subq_11.ds
       ) subq_15
       ON subq_7.metric_time__day = subq_15.metric_time__day
+      GROUP BY 1, 3
     ) subq_16
+---
+integration_test:
+  name: fill_nulls_with_0_multi_metric_query
+  description: Test a multi-metric query that fills nulls
+  model: SIMPLE_MODEL
+  metrics: ["bookings_fill_nulls_with_0", "views"]
+  group_by_objs: [{"name": "metric_time"}]
+  check_query: |
+    SELECT
+      COALESCE(subq_7.metric_time__day, subq_12.metric_time__day) AS metric_time__day
```

Comment:
Are there missing values in this date sequence for the bookings_fill_nulls_with metric? If so that's great. An output test would be nice but at least we can inspect the data frame from here and see it.

If not, it might be worth adding a dimension that has a gap for a given metric_time, dimension_value pair.

Reply:
Added a new test case & updated the source data to make sure there were some nulls filled for this case.
Will plan to add some output tests tomorrow!